pages/techgenius/tg-installation.md

```diff
@@ -55,7 +55,7 @@ Open your `Terminal`. We assume that [brew](http://brew.sh/) is already installe
 
 ## Ubuntu
 
-Note: You may [download this bash file](uploads/bashScripts/instructions.sh) to automate the installation process. Make sure that the bash file is in the directory where you would like to store the [ole--vagrant-vi](https://github.com/dogi/ole--vagrant-vi) repository. Go to terminal and use command `sudo sh instructions.sh`
+Note: You may [download this bash file](uploads/bashScripts/instructions.sh) to automate the installation process. The bash file can be downloaded by right clicking on the Raw button and clicking on "save link as" present in the top right corner.Make sure that the bash file is in the directory where you would like to store the [ole--vagrant-vi](https://github.com/dogi/ole--vagrant-vi) repository. Go to terminal and use command `sudo sh instructions.sh`
```

Comment:
I think @dogi wanted you to change the link (https://github.com/dogi/ole--vagrant-vi) to a relative link. A example would be the link right above "download this bash file".

I think the problem is that (https://github.com/dogi/ole--vargrant-vi) is in @dogi repository only. correct? so the way you have it now should be fine.